src/parseThresholds.ts

```diff
@@ -11,6 +11,7 @@ const regexFunctions = /functions\s*:\s*(\d+)/;
 
 const parseThresholds = async (vitestConfigPath: string): Promise<Thresholds> => {
   try {
+    // Input path may already be an absolute path (will shortcut 'path.resolve' call).
```

Comment:
And again, this comment describes what `path.resolve` does, so I do not really see the value to having it here - you mind removing it?

Reply:
Yeah, I fully agree with removing these two (also one in `parseJsonReports`), as it doesn't even really matter whether `path.resolve` will shortcut or not... (ie. no difference in execution).